pages/common/imapsync.md

```diff
@@ -0,0 +1,12 @@
+# imapsync
+
+> Email IMAP tool for syncing, copying and migrating email mailboxes
```

Comment:
The first line is usually a short snappy description, currently it doesn't make that much sense with the 2nd line dropped down. I know that technically it will be merged to a single line once rendered, but maybe just for readability?

Perhaps something like might be a bit shorter:
```
> A command line tool for managing incremental and recursive IMAP transfers.
> It allows syncing, copying and migrating email mailboxes between two imap servers, one way, and without duplicates.
```
Or I guess it could just be moved to a single line.

Reply:
Fixed!